src/components/SetupPanes/ForOffer.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import For from './For';
+import OfferPicker from '../FormComponents/OfferPicker.js';
+
+export default function ForAccount(props) {
+  let label = 'Offer ID';
+  let content = <OfferPicker
+    value={props.values['offer_id']}
+    onUpdate={(value) => {props.onUpdate('offer_id', value)}}
```

Comment:
Why not use `PositiveIntPicker` directly? `OfferPicker` doesn't seem to be doing anything special (other than setting a default placeholder).